modules/end-user-guide/partials/assembly_building-intellij-editor-images.adoc

```diff
@@ -0,0 +1,21 @@
+:parent-context-of-assembly_building-intellij-editor-images: {context}
+
+[id="assembly_building-intellij-editor-images_{context}"]
+= Building IntelliJ editor images
+
+:context: assembly_building-intellij-editor-images
+
+To build custom IntelliJ-based editor images, following methods are available:
```

Comment:
```
You can build a default or customized IntelliJ editor image:
```

Reply:
Are you sure you need `-based` after IntelliJ (IntelliJ-based)? I don't know the specifics here, but in general `IntelliJ` is usable as an adjective, so `IntelliJ editors` = `IntelliJ-based editors`.

Reply:
The user can different editors: IntelliJ-IDEA, Webstorm... They are all based on the IntelliJ framework.

Actually, looking at https://www.jetbrains.com/opensource/idea/ we should refer to the *IntelliJ Platform*.

Reply:
I found this "IntelliJ Platform-based products" here https://github.com/jetbrains

Reply:
> Even better: IDEs Based on the IntelliJ Platform﻿ ,https://plugins.jetbrains.com/docs/intellij/intellij-platform.html#ides-based-on-the-intellij-platform

I hope you add this link somewhere.

Reply:
Building an image for an IDE based on the IntelliJ Platform

Reply:
@max-cx,

*You can build a default or customized JetBrains IDE image:*

is not an improvement over:

*To build custom IntelliJ-based editor images*

In fact, it's worse. Overusing **you can** does not help readability. The orig. version is concise, to the point, and clear. The version starting with "you can" creates ambiguity and makes the instruction longer.